content/sauces/spaghetti-sauce.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Spaghetti Sauce
+---
+
+Ingredients:
+- Butter
+- Garlic
+- Italian spices
+- Flour
+- stock cubes or self made stock (vegetable or beef)
+- water (half a liter)
+- Sieved tomatoes
+
+Steps:
+1. Melt the butter in a pan
+2. finely mince garlic and add to butter together with some italian spices
+3. After heating for a bit, add some flour to make a roux
+4. in some of the water, dissolve the stock cubes
```

Comment:
wdym, in some of the water. What other water are we talking about?

Reply:
the half a liter of water isn't an exact amount, so it's best to start with some of the water. will clear this up when updating this MR with amounts